dbldatagen/data_generator.py

```diff
@@ -257,7 +274,8 @@ def explain(self, suppressOutput=False):
 
         output = ["", "Data generation plan", "====================",
```

Comment:
this `"===================="` is not aligned to the screen width. Use something like `rich` or `colorama` for proper printing.

Reply:
easier to fix code than add another library dependency. General philosophy to date has been to use what is in Databricks runtime release or in preparing it - hence preference for specific versions of numpy, pandas, use of Sphinx etc.

That way, we can take advantage of the product testing performed during the DB runtime release to ensure compatibility of different versions of different libraries etc without having to test ourselves. I.e assumption is that if DB runtime team have validated version compatibility, assumption is that it works